framework/src/components/storage/entities/account.js

```diff
@@ -450,7 +449,51 @@ class Account extends BaseEntity {
 			updateSet,
 		};
 
-		return this.adapter.executeFile(this.SQLs.update, params, {}, tx);
+		const membersPublicKeys = data.membersPublicKeys || [];
```

Comment:
If it doesn't exist, then don't initialize it with an empty array. It will be better if we add another condition in if statement like: 
`if (membersPublicKey && membersPublicKey.length > 0)`
I am requesting this because we call `update` function with `data` property including only the fields which are modified. Therefore, if `data.membersPublicKeys` doesn't exist, it is possible that account is registered multisignature account, but we are not updating the multisignature related infomration for that account.

Reply:
Done